test/unit/org/apache/cassandra/tcm/ClusterMetadataSerializerTest.java

```diff
@@ -47,16 +47,16 @@ public class ClusterMetadataSerializerTest
     public void serdeLatest()
     {
         DataOutputBuffer output = new DataOutputBuffer();
-        qt().forAll(new ClusterMetadataBuilder().build()).checkAssert(orFail(cm -> {
+        qt().forAll(Generators.toGen(new ClusterMetadataBuilder().build())).check(cm -> {
```

Comment:
accord `qt` will show the seed that failed when generators fail, but QuickTheories doesn't; making it hard to reproduce these type of errors.

I plan to migrate all tests off of QuickTheories *after* we merge to trunk, and these are cep-15-accord only tests....